content/downloads/linux/index.md

```diff
@@ -42,6 +42,21 @@ They depend on the following packages:
     sudo pacman -S sdl2 sdl2_net opusfile
 
 
+## <a name="steam"></a> Steam
+
+You can easily configure your DOS games on Steam to use dosbox-staging via
+[Boxtron](https://github.com/dreamer/boxtron) (community-developed
+Steam Play compatibility tool for DOS games).
```

Comment:
If your Steam library contains DOS games, we recommend using [Boxtron](https://github.com/dreamer/boxtron) to easily configure them to run using dosbox-staging, among other settings.  Boxtron is is a  community-developed Steam Play compatibility-tool for DOS games.